### ISSUE TITLE
impropper test for viewport size

### ISSUE THREAD

**Issue by masukomi** [vriteio/vrite#12]
see attached screenshot (full rez). This is NOT a small viewport, BUT I do use large fonts. ( 20, with min size set to 18 (advanced settings in firefox and chrome)). I'm asuming this is an accessibility issue related to how your measuring viewport size being somehow tied to the size of font characters, because otherwise this needs what i'd consider an unreasonably large viewport. 

also, wording should be changed because non-geeks don't know what a "viewport" is. 

![CleanShot 2023-06-23 at 09 57 02@2x](https://github.com/vriteio/vrite/assets/2829/a34f747f-1203-4e09-8342-eeffe3e87d1d)

**Comment by areknawo** [comment]
The viewport detection is based on `@media (min-width)` so I'm not sure if that's the problem. Besides, I'm afraid that, without some additional adjustment, the larger font could break the layout.

The best solution would be to get rid of this message entirely and just support smaller (phone-size) viewports.

**Comment by areknawo** [comment]
From v2023.7.12 Vrite now has support for mobile (i.e. small viewports). Some optimizations were also done for mid-sized viewports and touch inputs in general (e.g. tablets).

This might now be directly related to this issue and the layout might still break (more accessibility optimizations are required) but, at the very least, you should be able to see and interact with the UI.

Feel free to reopen in case you have any feedback/issues.
